olive/engine/packaging/packaging_generator.py

```diff
@@ -34,8 +34,11 @@ def generate_output_artifacts(
     if sum([len(f.nodes) if f.nodes else 0 for f in pf_footprints.values()]) == 0:
         logger.warning("No model is selected. Skip packaging output artifacts.")
         return
-    if packaging_config.type == PackagingType.Zipfile:
-        _generate_zipfile_output(packaging_config, footprints, pf_footprints, output_dir)
+    try:
+        if packaging_config.type == PackagingType.Zipfile:
+            _generate_zipfile_output(packaging_config, footprints, pf_footprints, output_dir)
+    except Exception as e:
```

Comment:
Seems we should avoid to catch generic `Exception` from previous discussion about to unify the exception in Olive.  Not sure in this case, do we know the detailed exception if possible?

Reply:
Good catch! This shouldn't be added here. Let me remove this.